components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/services/SessionManagementService.java

```diff
@@ -51,6 +50,7 @@ public boolean removeSession(String sessionId) {
      * @param sessionId
      * @return
      */
+    @Deprecated
```

Comment:
Please check whether adding a deprecated annotation to an admin service is supported?